.changeset/bright-colts-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+'website': minor
+---
+
+added prerelease badge
```

Comment:
commit empty changesets for website and storybook:

```
yarn changeset add --empty
```

Reply:
Done